frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { useQuery } from "@apollo/client";
+
+import "./TranslationPage.css";
+import { useParams } from "react-router-dom";
+import ProgressBar from "../translation/ProgressBar";
+import TranslationTable from "../translation/TranslationTable";
+import { StoryLine } from "../translation/Autosave";
+import { GET_STORY_AND_TRANSLATION_CONTENTS } from "../../APIClients/queries/StoryQueries";
+import CommentsPanel from "../review/CommentsPanel";
+
+type ReviewPageProps = {
+  storyIdParam: string | undefined;
+  storyTranslationIdParam: string | undefined;
+};
+
+type Content = {
+  id: number;
+  lineIndex: number;
+  content: string;
+  status: string;
+};
+
+const ReviewPage = () => {
+  const {
+    storyIdParam,
+    storyTranslationIdParam,
+  } = useParams<ReviewPageProps>();
+
+  const storyId = +storyIdParam!!;
+  const storyTranslationId = +storyTranslationIdParam!!;
+  const [translatedStoryLines, setTranslatedStoryLines] = useState<StoryLine[]>(
+    [],
+  );
+  const [numTranslatedLines, setNumTranslatedLines] = useState(0);
+
+  useQuery(GET_STORY_AND_TRANSLATION_CONTENTS(storyId, storyTranslationId), {
+    fetchPolicy: "cache-and-network",
+    onCompleted: (data) => {
+      const storyContent = data.storyById.contents;
+      const translatedContent = data.storyTranslationById.translationContents;
+
+      setNumTranslatedLines(data.storyTranslationById.numTranslatedLines);
+
+      const contentArray: StoryLine[] = [];
+      storyContent.forEach(({ content, lineIndex }: Content) => {
+        contentArray.push({
+          lineIndex,
+          originalContent: content,
+          status: "Default",
+        });
+      });
+
+      contentArray.sort((a, b) => a.lineIndex - b.lineIndex);
+
+      translatedContent.forEach(({ id, content, lineIndex }: Content) => {
+        const arrIndex = lineIndex - contentArray[0].lineIndex;
```

Comment:
Think this is unnecessary, without pagination `lineIndex === contentArray[0].lineIndex` always